packages/protocol/src/data/inscription-data.ts

```diff
@@ -47,3 +48,9 @@ export interface InscriptionSaveData {
 }
 
 export type ElementType = InscriptionType['id'];
+
+type TaskCaseSchemaKeys = 'name' | 'description' | 'category' | 'expriy';
+
+type EmailSchemaKeys = 'subject' | 'from' | 'to' | 'replyTo' | 'cc' | 'bcc';
```

Comment:
maybe there is another typescript magic way to extract these keys from the properties of the generated inscription objects?

![header-properties](https://github.com/axonivy/inscription-client/assets/15085693/efb961d8-84f4-4b64-a323-64c80a40735e)

Reply:
I try, but will be placed in a follow up PR.